src/lib/customTraceType.js

```diff
@@ -97,6 +97,6 @@ export function traceTypeToPlotlyInitFigure(traceType, gl = '') {
         type: 'cone',
       };
     default:
-      return {type: traceType};
+      return {type: traceType ? traceType : 'scatter'};
```

Comment:
This is actually problematic

Reply:
The scatter case should set the same options as at the top otherwise there are two “scatter” paths